### ISSUE TITLE
Wrong profile_pic_url_hd and biography when passing SESSION_ID

### ISSUE THREAD

**Issue by marco97pa** [chris-greening/instascrape#90]
## Describe the bug
If we pass a valid **SESSION_ID** cookie to the scraper, the values **profile_pic_url_hd** and **biography** of the Profile are wrong.

### Code
Run this code by providing a valid **instagram_sessionid** 
```
from instascrape import *

instagram_sessionid = 'xxxxx'
headers = {"user-agent": "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Mobile Safari/537.36 Edg/87.0.664.57",
"cookie": f"sessionid={instagram_sessionid};"}

profile = Profile("roses_are_rosie")
profile.scrape(headers=headers)
print(profile.profile_pic_url_hd)
print(profile.biography)
```

### Expected output
We expect the output to print the profile pic url and biography of [@roses_are_rosie](https://www.instagram.com/roses_are_rosie/):
```
https://scontent-fco1-1.cdninstagram.com/v/t51.2885-19/s320x320/120911233_113942167015019_7757793538086741578_n.jpg?_nc_ht=scontent-fco1-1.cdninstagram.com&_nc_ohc=vLiuePti8-oAX81KArw&tp=1&oh=6347fbcf84fa8021ca0f427c4b355573&oe=604ABB41
ROSÉ
```

### Output
Instead we got the profile pic url and the biography of the login account of the provided instagram_sessionid.
In my specific case, it is printing the data of [@puntiburraco](https://www.instagram.com/puntiburraco/)
Please be aware that the output depends on the login account you are using to scrape, but it's always wrong.
```
https://scontent-fco1-1.cdninstagram.com/v/t51.2885-19/s320x320/66284490_468664600594844_7307310439468105728_n.jpg?_nc_ht=scontent-fco1-1.cdninstagram.com&_nc_ohc=y6o_TmT3c6AAX9PGXzO&tp=1&oh=240222f677c66049485e2edad1d76c77&oe=604E1EC2
App Android
Segnapunti per le tue partite di Burraco
♠️♥️♣️♦️🃏
```

**Tested on:**
 - OS: Windows 10 x64
 - Python 3.8
 - instascrape 2.0.2

**Additional context**
I understand that the latest changes of Instagram broke the library but we need more documentation on how to handle this new changes.

**Comment by marco97pa** [comment]
Seems the same bug as #82, which was supposed to be fixed

**Comment by marco97pa** [comment]
Fixed as **instascrape 2.1.2**
